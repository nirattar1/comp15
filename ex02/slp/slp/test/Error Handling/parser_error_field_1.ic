class Stam {
	int i
	void foo() {
		i = i+1;
	}
}