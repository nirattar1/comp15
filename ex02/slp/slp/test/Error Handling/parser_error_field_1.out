3: Syntax error: at token void